.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "legion"]
 	path = legion
-	url = ../../StanfordLegion/legion.git
+	url = https://gitlab.com/StanfordLegion/legion.git
```

Comment:
That is bad, because it will break mirroring into the internal gitlab.

Just ask @elliottslaughter and @streichler to mirror from gitlab.com more regularly.